client/allocrunner/network_hook.go

```diff
@@ -70,7 +71,7 @@ func (h *networkHook) Prerun() error {
 		h.setter.SetNetworkIsolation(spec)
 	}
 
-	if err := h.networkConfigurator.Setup(h.alloc, spec); err != nil {
+	if err := h.networkConfigurator.Setup(context.Background(), h.alloc, spec); err != nil {
```

Comment:
These added contexts are used when running cni commands. Should we plumb in context with a timeout or some other parent context here?